api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -25,6 +41,37 @@ trait HttpApi extends HmdaApiProtocol {
           ToResponseMarshallable(status)
         }
       }
-    }
+    } ~
+      path("upload") {
+        post {
```

Comment:
Why not the [`fileUpload`](http://doc.akka.io/docs/akka/current/scala/http/routing-dsl/directives/file-upload-directives/fileUpload.html) directive?